libs/langchain/langchain/vectorstores/surrealdb.py

```diff
@@ -0,0 +1,434 @@
+import asyncio
+from typing import (
+    Any,
+    Iterable,
+    List,
+    Optional,
+    Tuple,
+)
+
+from surrealdb import Surreal
+
+from langchain.docstore.document import Document
+from langchain.schema.embeddings import Embeddings
+from langchain.schema.vectorstore import VectorStore
+
+
+class SurrealDBStore(VectorStore):
```

Comment:
this should be imported into `langchain/vectorstores/__init__.py`

Reply:
Thanks. this is now done.